sdk/communication/Azure.Communication.JobRouter/samples/README.md

```diff
@@ -0,0 +1,61 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-communication-services
+# - azure-communication-services-job-router
```

Comment:
We should consider uncommenting this or removing it.

Reply:
This has been removed